xml/containerized-kubernetes.xml

```diff
@@ -87,14 +110,14 @@
   <para>
    The &rook; orchestrator uses configuration files in YAML format called
    <emphasis>manifests</emphasis>. The manifests you need are included in the
-   <package>rook-k8s-yaml</package> RPM package. Install it by running
+   <package>rook-k8s-yaml</package> RPM package. You can find this package
```

Comment:
you forgot to remove "You can find this package in <filename>/usr/share/k8s-yaml" it should be " That package is in SES6 repos"